server/lib/genome/importers/api_importers/civic/importer.rb

```diff
@@ -55,39 +55,40 @@ def create_gene_claims
             end
           end
 
-          def create_entries_for_evidence_item(ei, dc, gc)
-            ic = create_interaction_claim(gc, dc)
-            if ei.source.citation_id.present? && ei.source.source_type == 'PubMed'
-              create_interaction_claim_publication(ic, ei.source.citation_id)
+          def create_entries_for_evidence_item(ev_item, drug_claim, gene_claim)
+            ic = create_interaction_claim(gene_claim, drug_claim)
+            if ev_item.source.citation_id.present? && ev_item.source.source_type == 'PUBMED'
+              create_interaction_claim_publication(ic, ev_item.source.citation_id)
             end
-            create_interaction_claim_link(ic, "EID#{ei.id}", "https://civicdb.org/evidence/#{ei.id}")
+            create_interaction_claim_link(ic, "ev_itemD#{ev_item.id}", "https://civicdb.org/evidence/#{ev_item.id}")
```

Comment:
Is the capital `D` supposed to remain here after `ev_item`?

Reply:
not intentional!